### DISCUSSION TITLE
Change Theme on Mac

### DISCUSSION THREAD

**Discussion by untropee** [brackets-cont/brackets#235] [Feature Request]
Hello. I downloaded this theme https://github.com/Brackets-Themes/EasyDark
Help me please, how I can install this into Brackets on Mac OS?

**Comment by pablosed** [comment]
not the dev - but I installed it easily by going to File>extension manager, then clicking on "install from URL" on bottom left of the window, then pasting your link into there. Then when I went to View>Themes it was listed in the drop down and I can activate. Good luck!
